src/main/java/org/fcrepo/spec/testsuite/AbstractTest.java

```diff
@@ -55,26 +54,43 @@
 /**
  * A crud base class for common crud functions.
  *
- * @author Daniel Berntein
+ * @author dbernstein
  */
 public class AbstractTest {
 
     protected PrintStream ps;
 
+    protected final String adminUsername;
+    protected final String adminPassword;
     protected final String username;
     protected final String password;
 
     /**
      * Constructor
      *
+     * @param adminUsername admin password
+     * @param adminPassword admin username
      * @param username username
      * @param password password
      */
-    public AbstractTest(final String username, final String password) {
+    public AbstractTest(final String adminUsername, final String adminPassword, final String username,
+                        final String password) {
+        this.adminUsername = adminUsername;
+        this.adminPassword = adminPassword;
         this.username = username;
         this.password = password;
     }
 
+    /**
+     * Constructor
+     *
+     * @param username username
+     * @param password password
+     */
+    public AbstractTest(final String username, final String password) {
+        this(username, password, username, password);
```

Comment:
It may be less confusing if instead the admin username and password be set to `null`.